plugin/builtin/buildbaron/build_baron.go

```diff
@@ -33,18 +39,15 @@ const (
 	maxNoteSize = 16 * 1024 // 16KB
 )
 
+type bbProject evergreen.BuildBaronProject
```

Comment:
Can you do away with this alias? We generally don't alias things like this in our codebase